### DISCUSSION TITLE
Log Dumps wanted?

### DISCUSSION THREAD

**Discussion by justbruce** [esphome-econet/esphome-econet#214] [General]
I've been reviewing my logs and have noticed that my system is producing output that I haven't noticed on this site 

 First, there are 2 additional hardware SRC and DST address (0x1c0, 0x600) and several new datapoints

HWELMODL
HWCONFIG
ODU__KWH
MAXCLAMP

[I'm not super familiar with the code base, but quick seaches of the repository don't show any hits ]

Would you like a DEBUG level log file?  If so, what is the best way to get it to you, while still kinda maintaining my privacy?

**Comment by barndawgie** [comment]
The most personal thing in there would be your Furnace's serial number so I wouldn't worry too much about that.

**Comment by Daniel-dev22** [comment]
> I've been reviewing my logs and have noticed that my system is producing output that I haven't noticed on this site 
> 
>  First, there are 2 additional hardware SRC and DST address (0x1c0, 0x600) and several new datapoints
> 
> HWELMODL
> HWCONFIG
> ODU__KWH
> MAXCLAMP
> 
> [I'm not super familiar with the code base, but quick seaches of the repository don't show any hits ]
> 
> Would you like a DEBUG level log file?  If so, what is the best way to get it to you, while still kinda maintaining my privacy?
> 
> 

I knew of these objects but my unit doesn't support it so yes seeing the logs will be very helpful. 0x1c0 is the furnace I believe.

**Comment by justbruce** [comment]
This is about 45 seconds worth of logging.  1222 lines.  Let me know if you need more


[justbruce-hvac-log.txt](https://github.com/esphome-econet/esphome-econet/files/13490820/justbruce-hvac-log.txt)

**Reply by Daniel-dev22** [reply]
> This is about 45 seconds worth of logging.  1222 lines.  Let me know if you need more
> 
> 
> [justbruce-hvac-log.txt](https://github.com/esphome-econet/esphome-econet/files/13490820/justbruce-hvac-log.txt)
> 

Longer would be helpful.

Can you explain your system setup as well?

Heatpump? Furnance? Both? etc.

**Reply by justbruce** [reply]
How much longer?  Currently at about 3.5 minutes worth

Dual-fuel setup. 
-3 Stage, 4-ton Heat pump
- 98% furnace
-700 series Econet thermostat

**Reply by Daniel-dev22** [reply]
> How much longer?  Currently at about 3.5 minutes worth
> 
> Dual-fuel setup. 
> -3 Stage, 4-ton Heat pump
> - 98% furnace
> -700 series Econet thermostat

> How much longer?  Currently at about 3.5 minutes worth

That should be good 

> Dual-fuel setup. 
> -3 Stage, 4-ton Heat pump
> - 98% furnace
> -700 series Econet thermostat

Is your heatpump communicating? I assume so based on the logs?

Any idea how old the heatpump is/model now?

**Reply by justbruce** [reply]
Heat pump is a communicating model.  

Unit was installed in July.  

I can grab the model number later today if that is helpful

**Reply by justbruce** [reply]
Model Number RP1748AJVCA

**Comment by justbruce** [comment]
[justbruce-hvac-log2.txt](https://github.com/esphome-econet/esphome-econet/files/13491001/justbruce-hvac-log2.txt)

7 minutes of logging.  ~11k lines

**Reply by justbruce** [reply]
just noticed that there is also a SRC/DST address of  '0xf1' ..  Not sure which component that is

**Reply by justbruce** [reply]
Also, just on a hunch, unplugged my thermostat and plugged it back in.  I was hoping to see something new.  I have the log fil for that if anyone is interested.

**Reply by Daniel-dev22** [reply]
> just noticed that there is also a SRC/DST address of  '0xf1' ..  Not sure which component that is

That's a broadcast address.

**Reply by Daniel-dev22** [reply]
> Also, just on a hunch, unplugged my thermostat and plugged it back in.  I was hoping to see something new.  I have the log fil for that if anyone is interested.
> 
> 

Doesn't hurt to see. Can you share that?

**Comment by justbruce** [comment]
There are two files attached - the raw log from the disconnected thermostat, and then a file of some of the CharCode decoded.

  Interestingly, the thermostat seems to ask the heat pump for the heat pump's error codes on startup.  Don't think that it does that with the furnace.  Or maybe I just need to power cycle the furnace with logging on...... hmm....

  Anyways, hope this is helpful


[justbruce-hvac-log-3.txt](https://github.com/esphome-econet/esphome-econet/files/13555240/justbruce-hvac-log-3.txt)
[justbruce-hvac-charcodes-decode.txt](https://github.com/esphome-econet/esphome-econet/files/13555241/justbruce-hvac-charcodes-decode.txt)
